cli/valet.php

```diff
@@ -86,13 +86,36 @@
 
         Configuration::updateKey('tld', $tld);
 
-        Site::resecureForNewTld($oldTld, $tld);
+        Site::resecureForNewConfiguration(['tld' => $oldTld], ['tld' => $tld]);
         PhpFpm::restart();
         Nginx::restart();
 
         info('Your Valet TLD has been updated to ['.$tld.'].');
     }, ['domain'])->descriptions('Get or set the TLD used for Valet sites.');
 
+    /**
+     * Get or set the loopback address currently being used by Valet.
+     */
+    $app->command('loopback [loopback]', function ($loopback = null) {
+        if ($loopback === null) {
+            return output(Configuration::read()['loopback']);
+        }
+
+        $oldLoopback = Configuration::read()['loopback'];
+
+        Configuration::updateKey('loopback', $loopback = trim($loopback, '.'));
```

Comment:
Question: Why the `trim(x, '.')` here?

Reply:
I'll will replace with input validation.